react/global.css

```diff
@@ -212,6 +216,10 @@ body.vtex-filter-popup-open {
   .vtex-gallery--two-columns {
     grid-template-columns: repeat(2, 1fr);
   }
+
+  body.vtex-search-result-sidebar-open {
+    overflow: hidden;
```

Comment:
Use Tachyons's `overflow-hidden` instead

Reply:
@luciannojunior better not to, this way we would have an idea of who added the `overflow: hidden` to the body, making debugging easier if we have a problem with this in the future.